### PR TITLE
LPS-121441: The set values of "Select Structure Field" are displayed for a moment before disappearing.

### PR DESCRIPTION
**Pull request by ces-totrinh**
Hi @hongvo2308
I update my work for the ticket: https://issues.liferay.com/browse/LPS-121441
Please help me review it and give me your comments.
Thanks.

Note: If you have any questions. Please let me know.

### CODE REVIEW

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/init.jsp [hunk 1]
```diff
@@ -74,6 +74,8 @@ page import="com.liferay.portal.kernel.dao.search.ResultRow" %><%@
 page import="com.liferay.portal.kernel.dao.search.SearchContainer" %><%@
 page import="com.liferay.portal.kernel.exception.NoSuchGroupException" %><%@
 page import="com.liferay.portal.kernel.exception.NoSuchModelException" %><%@
+page import="com.liferay.portal.kernel.json.JSONObject" %><%@
+page import="com.liferay.portal.kernel.json.JSONUtil" %><%@
 page import="com.liferay.portal.kernel.language.LanguageUtil" %><%@
 page import="com.liferay.portal.kernel.log.Log" %><%@
 page import="com.liferay.portal.kernel.log.LogFactoryUtil" %><%@
```
Comment (ces-hongvo) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/init.jsp:81]: @tototrinh 
The space delete is from the source formatter, right ?

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/select_structure_field.jsp [hunk 2]
```diff
@@ -156,7 +179,6 @@ portletURL.setParameter("eventName", eventName);
 
 	var submitForm = function (applyButton) {
 		var result = Util.getAttributes(applyButton, 'data-');
-
```
Comment (ces-hongvo) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/select_structure_field.jsp]: @tototrinh 
Why do you remove this ?

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/select_structure_field.jsp [hunk 3]
```diff
@@ -92,10 +92,6 @@ portletURL.setParameter("eventName", eventName);
 					ddmField.setDefaultLocale(themeDisplay.getLocale());
 					ddmField.setDDMStructureId(ddmStructureId);
 					ddmField.setName(name);
-
-					if (name.equals(ddmStructureFieldName)) {
-						ddmField.setValue(themeDisplay.getLocale(), ddmStructureFieldValue);
-					}
```
Comment (ces-hongvo) [inline modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/select_structure_field.jsp]: If these statements are not related to the current bugs please leave them unchanged.

**Comment by ces-totrinh** [comment]
Hi @hongvo2308 ,
Thanks for your feedback. I updated the code. Please take a look again. Thanks

File: modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/init.jsp (hunk 1 above)
Comment (ces-totrinh) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/init.jsp:81]: These are blank characters. I just added 2 imports.

Comment (ces-hongvo) [reply modules/apps/asset/asset-publisher-web/src/main/resources/META-INF/resources/init.jsp:81]: sorry, my mistake.

**Review by hongvo2308 — APPROVED** [review]

**Comment by ces-hongvo** [comment]
https://github.com/holatuwol/liferay-portal/pull/227
